doc/containers.md

```diff
@@ -172,7 +172,7 @@ host\_name              | string    | randomly assigned | no        | bridged, p
 hwaddr                  | string    | randomly assigned | no        | all                           | -             | The MAC address of the new interface
 mtu                     | integer   | parent MTU        | no        | all                           | -             | The MTU of the new interface
 parent                  | string    | -                 | yes       | physical, bridged, macvlan    | -             | The name of the host device or bridge
-vlan                    | integer   | -                 | no        | macvlan                       | network\_vlan | The VLAN ID to attach to
+vlan                    | integer   | -                 | no        | macvlan, physical             | network\_vlan | The VLAN ID to attach to
```

Comment:
As I am thinking about what to write, I realize that this issue is unrelated to VLAN. What you describe is related to "physical" vs "macvlan". If you use "physical" nictype then only one container can be attached to the device but network packets with destination MAC different from NIC MAC can reach the container and then be forwarded inside the container through a switch or to nested containers. If you use "macvlan" nictype then several containers can be attached to the device with drawbacks... packets with different MAC destination won't be forwarded inside the container.
As this is totally unrelated to VLAN, I propose to not include this documentation improvement in this pull request.
I will create a specific issue for "macvlan vs physical documentation improvement" and then submit a new patch.
Does it make sens?